File: src/scala/classess/Person.scala
package scala.classess

import java.util.Date

/**
 * Created on 16.04.16.
 * @author evgen
 */
class Person {

  private var age = 0
  val time = new Date

}

object Person {

  def main(args: Array[String]) {
    val person:Person = new Person
    println(person)
    println(person.age)
    person.age = 29
    println(person.age)
  }
}

File: src/scala/serializable/Person.java
package scala.serializable;

import java.io.Serializable;

/**
 * Created on 26.05.16.
 *
 * @author evgen
 */
public class Person implements Serializable {

    private long id;
    private String firstName;

    public Person(long id, String firstName) {
        this.id = id;
        this.firstName = firstName;
    }

    public String getFirstName() {
        return firstName;
    }

    public long getId() {
        return id;
    }
}
